src/pages/prompt/index.tsx

```
@@ -1,75 +1,60 @@
-              <h2 className="card-title line-clamp-1">{prompt.title}</h2>
-              <p className="text-sm -mt-2 line-clamp-1">{prompt.prompt}</p>
-              <div className="card-actions justify-end mt-2">
+      <div className="p-4 flex justify-center">
+        <ul className="list bg-base-200 rounded-box w-xl">
+          {promptList && promptList.length <= 0 && (
+            <li className="list-row flex text-center">
+              <p className="w-full">No prompt was found.</p>
+            </li>
+          )}
+          {promptList?.map((prompt) => (
+            <li className="list-row flex">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)